apps/web/src/xwagmi/hooks/useXDisconnectAll.ts

```diff
@@ -0,0 +1,17 @@
+import { useCallback } from 'react';
+import { useXDisconnect } from './useXDisconnect';
+
+export function useXDisconnectAll() {
+  const xDisconnect = useXDisconnect();
+
+  const disconnectAll = useCallback(async () => {
+    await Promise.all([
```

Comment:
let's think about a good way to disconnect all without hardcoding.

Reply:
yes, it's temporary, let's do it after merged